parts/k8s/kubernetesinstalls.sh

```diff
@@ -175,34 +175,46 @@ function installImg() {
     retrycmd_get_executable 120 5 $img_filepath "https://acs-mirror.azureedge.net/img/img-linux-amd64-v0.4.6" ls || exit $ERR_IMG_DOWNLOAD_TIMEOUT
 }
 
-function pullHyperkube() {
-    retrycmd_if_failure 60 1 1200 img pull $HYPERKUBE_URL || exit $ERR_K8S_DOWNLOAD_TIMEOUT
-    img unpack -o "/home/rootfs-${KUBERNETES_VERSION}" $HYPERKUBE_URL
-    path=$(find /home/rootfs-${KUBERNETES_VERSION} -name "hyperkube")
+function extractHyperkube() {
```

Comment:
This function is the one doing the extraction really so renamed to extractHyperkube